compliance/src/main/java/bio/terra/pearl/compliance/model/GithubAccount.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.pearl.compliance.model;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Collection;
+
+@Getter @Setter @SuperBuilder @NoArgsConstructor
+public class GithubAccount extends VantaObject {
+
+    @Override
+    public String getIntegrationId() {
+        return "github";
+    }
+
+    private String displayName, owner, description, accountName;
+
+    private boolean isDeactivated, inScope;
+
+    @Override
+    public boolean shouldBeInScope(Collection<PersonInScope> peopleInScope) {
+        return peopleInScope.stream().anyMatch(personInScope -> accountName.equalsIgnoreCase(personInScope.getGitUser()));
+    }
+
+    @Override
+    public String getSimpleId() {
+        return getDisplayName();
+    }
+
+    @Override
```

Comment:
FWIW Lombok provides a `@ToString` annotation you can put at the top of a class that will generate something very much like this